docs/resources/instance_ip.md

```diff
@@ -29,6 +33,7 @@ In addition to all above arguments, the following attributes are exported:
 ~> **Important:** Instance IPs' IDs are [zoned](../guides/regions_and_zones.md#resource-ids), which means they are of the form `{zone}/{id}`, e.g. `fr-par-1/11111111-1111-1111-1111-111111111111`
 
 - `address` - The IP address.
+- `prefix` - The IP Prefix.
```

Comment:
v4 or v6 ?

Reply:
It will be either v4 or v6 depending of the IP type